runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -207,4 +207,19 @@ default int slaughter()
 		description = ""
 	)
 	void slaughter(int slaughter);
+
+	@ConfigItem(
+			keyName = "doubleTroubleEnabled",
+			name = "",
+			description = "",
+			hidden = true
+	)
+	default boolean doubleTroubleEnabled() { return false; }
```

Comment:
expand the code block and place the { on a new line

Reply:
Fixed.